junit-jupiter-api/src/main/kotlin/org/junit/jupiter/api/Assertions.kt

```diff
@@ -0,0 +1,93 @@
+/*
```

Comment:
The compiler will generate a class called `AssertionsKt` to store these methods inside of. Is this acceptable? We can rename the generated class if necessary.

Reply:
I think that's fine

Reply:
Cool!